### PR TITLE
API and General Documentation

### PR DESCRIPTION
**Pull request by FirestarJes**
Started creating general and API documentation for the Market Participant domain

### CODE REVIEW

**Review by xnetsrak — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,4 +2,253 @@
...
+
+### GET:/Organization/ID
+
+```organization/{organizationId:guid}```<br />
+*Returns an organization with the specified id, if it exists.*
+
+### POST:/Organization
+
+```organization/```<br />
+*Returns and organization with the specified id, if it exists.*
```
Comment (michaelmajgaard) [inline README.md:45]: Creates a new organization with...

File: README.md [hunk 2]
```diff
@@ -2,4 +2,253 @@
...
+```json
+{
+  "name": "string",
+  "category": "string",
+  "email": "string",
+  "phone": "string"
+}
+```
+
+### DELETE:/Organization/Contact
```
Comment (michaelmajgaard) [inline README.md:171]: Contacts are assigned to actors now. Organisations no longer have any contacts.

File: README.md [hunk 3]
```diff
@@ -2,4 +2,253 @@
...
+  "category": "string",
+  "email": "string",
+  "phone": "string"
+}
+```
+
+### DELETE:/Organization/Actor/Contact
+
+```organization/{organizationId:guid}/contact/{contactId:guid}```<br />
+*Deletes a contact from the specified organization, if it exists*
```
Comment (michaelmajgaard) [inline README.md:202]: 'from the specified actor in the specified organisation`

File: README.md [hunk 4]
```diff
@@ -2,4 +2,253 @@
...
+```
+
+---
+---
+
+## Integration Events
+
+Integration events are a way for interested parties to know when data changes in the Market Participant domain and be informed of these changes.
+
+Integration events are published to a servicebus topic, where everyone who are interested can listen. What you do with these events are. A description of how to subscribe to these events are [here](#Subscribe-to-integration-events)
```
Comment (michaelmajgaard) [inline README.md:233]: 'What you do with these events are .. ?' incomplete sentence

Comment (michaelmajgaard) [inline README.md:233]: replace 'are [here..' with 'is found [here..'

Comment (FirestarJes) [reply README.md]: Fixed

Comment (FirestarJes) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (FirestarJes) [reply README.md]: Removed section with organization contacts

File: README.md (hunk 2 above)
Comment (FirestarJes) [reply README.md]: from the specified actor in the specified organisation

File: README.md (hunk 1 above)
Comment (FirestarJes) [reply README.md]: fixed

**Review by michaelmajgaard — APPROVED** [review]

**Pull request merged**